### PR TITLE
fix(scss): fix fouc loader

### PR DESCRIPTION
**Pull request by saas786**
Removes un-necessary css and also fixes loader issue.

### CODE REVIEW

File: packages/cxl-lumo-styles/scss/loading.css [hunk 1]
```diff
@@ -15,16 +15,14 @@ body::before {
...
-  align-items: center;
-  justify-content: center;
   background-color: #fff;
 }
 
 /**
  * @see https://developers.google.com/web/fundamentals/performance/rendering/stick-to-compositor-only-properties-and-manage-layer-count
  */
 body[unresolved] {
+  position: relative;
```
Comment (lkraav) [inline packages/cxl-lumo-styles/scss/loading.css:25]: Do we have any theory of why exactly relative would win over absolute here :thinking:

File: packages/cxl-lumo-styles/scss/loading.css [hunk 2]
```diff
@@ -15,16 +15,14 @@ body::before {
   right: 0;
   bottom: 0;
   left: 0;
-  display: flex;
-  align-items: center;
-  justify-content: center;
```
Comment (lkraav) [inline packages/cxl-lumo-styles/scss/loading.css:20]: Relative with edge property 0-s automatically puts spinner in middle, really?

Comment (saas786) [reply packages/cxl-lumo-styles/scss/loading.css]: > Relative with edge property 0-s automatically puts spinner in middle, really?

It was a mistake, rolled back.

File: packages/cxl-lumo-styles/scss/loading.css (hunk 1 above)
Comment (saas786) [reply packages/cxl-lumo-styles/scss/loading.css:28]: > Do we have any theory of why exactly relative would win over absolute here 🤔

Default position is static, which doesn't render z-index, where if we set it to relative position, z-index is considered in rendering of dom. And as z-index of child is relative to parent z-index, so maybe that's why it works with position: relative.

Also changed pseudo `::before` position to fixed, now it overlays the content, all the time.

**Comment by lkraav** [comment]
Commit message can't be feat, if explanation says fix..

**Comment by saas786** [comment]
> Commit message can't be feat, if explanation says fix..

I was under the impression to not use "Fix:" in commit messages, instead rather use "Feat:"? 
Will change it.

**Comment by lkraav** [comment]
> > Commit message can't be feat, if explanation says fix..
> 
> I was under the impression to not use "Fix:" in commit messages, instead rather use "Feat:"? 

From where? I've never seen such a policy anywhere.

**Comment by saas786** [comment]
> > > Commit message can't be feat, if explanation says fix..
> > 
> > 
> > I was under the impression to not use "Fix:" in commit messages, instead rather use "Feat:"?
> 
> From where? I've never seen such a policy anywhere.

I must have been mistaken :), will use fix moving forward.

File: packages/cxl-lumo-styles/scss/loading.css (hunk 1 above)
Comment (saas786) [reply packages/cxl-lumo-styles/scss/loading.css:28]: > Was there ever a screencast anywhere that demonstrates what problem this whole commit is fixing?
> 
> We haven't heard loading screen being broken in any way for a year now.

Yes, this thread: https://cxlworld.slack.com/archives/C01JABH8AHX/p1615215936047400?thread_ts=1615210993.043300&cid=C01JABH8AHX

[screencast-loader-2021.03.11-19_00_38.webm](https://user-images.githubusercontent.com/1642796/177554328-7f4a1119-af9e-4521-9866-1322c4ff7a72.webm)

Though it was happening on jobs website only, but applying this PR, does seem to resolve the issue.

**Comment by pawelkmpt** [comment]
Obsolete

**Pull request closed without merging**